File: UnitTestProjectVB1/UnitTest1.vb
Imports System.Text
Imports Microsoft.VisualStudio.TestTools.UnitTesting

<TestClass()> Public Class UnitTest1

    <TestMethod()> Public Sub TestMethod1()

        'Arrange
        Dim expectiveValue = "Hello"

        'Act
        Dim frm As New WindowsAppVB1.DemoHelloForm
        Dim actualValue = frm.SayHello()

        'Assert
        Assert.AreEqual(expectiveValue, actualValue)



    End Sub

End Class

File: UnitTestProject1/UnitTest1.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System.Windows.Forms;

namespace UnitTestProject1
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            //Arrange
            string expectiveValue = "Hello";

            //Act
            WindowsFormsApp1.Demo.DemoHelloForm frm = new WindowsFormsApp1.Demo.DemoHelloForm();
            string actualValue = frm.SayHello();

            //Assert
            Assert.AreEqual(expectiveValue, actualValue);

        }
    }
}
